mathics/builtin/datentime.py

```diff
@@ -96,6 +98,12 @@ def total_seconds(td):
 else:
     total_seconds = timedelta.total_seconds
 
+SymbolAbsoluteTime = Symbol("AbsoluteTime")
+SymbolDateObject = Symbol("DateObject")
+SymbolDateString = Symbol("DateString")
+SymbolGregorian = Symbol("Gregorian")
+SymbolPause = Symbol("Pause")
+
```

Comment:
I think this is a better way to handle Symbols that largely appear here and won't in other places (as opposed to adding them to systemsymbols). 

For those things though that are common systemsymbols is fine.